android/java/org/chromium/chrome/browser/crypto_wallet/util/PendingTxHelper.java

```diff
@@ -60,14 +62,7 @@ public PendingTxHelper(TxService txService, AccountInfo[] accountInfos, boolean
         mTransactionInfoLd = _mTransactionInfos;
         mSelectedPendingRequest = _mSelectedPendingRequest;
         mHasNoPendingTxAfterProcessing = _mHasNoPendingTxAfterProcessing;
-    }
 
-    public PendingTxHelper(
-            TxService txService,
-            AccountInfo[] accountInfos,
-            boolean returnAll,
-            boolean shouldObserveTxUpdates) {
-        this(txService, accountInfos, returnAll);
```

Comment:
Constructor `PendingTxHelper(TxService, AccountInfo[], boolean, boolean)` removed as it was unused.